### PR TITLE
Remove bulk sale

### LINKED ISSUE #98: remove bulk sale code

### PR DESCRIPTION
**Pull request by ZenVoich**
fixes #98

### CODE REVIEW

**Comment by ZenVoich** [comment]
remove `amount` from `reserve`

File: src/Sale/types.mo [hunk 1]
```diff
@@ -102,7 +102,7 @@ module {
     whitelistTime : Time;
     whitelist : Bool;
     totalToSell : Nat;
-    bulkPricing : [(Nat64, Nat64)];
+    bulkPricing : [(Nat64, Nat64)]; // always [(1, <price>)]
```
Comment (letmejustputthishere) [inline src/Sale/types.mo:105]: remove bulkPricing entirely

File: src/main.mo [hunk 2]
```diff
@@ -394,10 +394,10 @@ shared ({ caller = init_minter }) actor class Canister(cid : Principal, initArgs
     _Sale.enableSale(caller);
   };
 
-  public func reserve(amount : Nat64, quantity : Nat64, address : SaleTypes.AccountIdentifier, _subaccountNOTUSED : SaleTypes.SubAccount) : async Result.Result<(SaleTypes.AccountIdentifier, Nat64), Text> {
+  public func reserve(amount : Nat64, quantityNOTUSED : Nat64, address : SaleTypes.AccountIdentifier, _subaccountNOTUSED : SaleTypes.SubAccount) : async Result.Result<(SaleTypes.AccountIdentifier, Nat64), Text> {
```
Comment (letmejustputthishere) [inline src/main.mo:397]: remove unused arguments as we control the sale frontend

Comment (letmejustputthishere) [reply src/main.mo:397]: also gret rid of amount

**Review by letmejustputthishere — CHANGES_REQUESTED** [review]

**Review by letmejustputthishere — APPROVED** [review]

**Pull request merged**